src/components/LoginForm/index.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { Fieldset, InputText, Button } from '@edx/paragon';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+import LoadingSpinner from '../LoadingSpinner';
+import Footer from '../Footer';
+
+
+// TODO: redirect forgot password to correct site
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+    };
+    this.handleEmailChange = this.handleEmailChange.bind(this);
+    this.handlePasswordChange = this.handlePasswordChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleEmailChange(event) {
+    this.setState({
+      email: event.trim(),
+    });
+  }
+
+  handlePasswordChange(event) {
+    this.setState({
+      password: event.trim(),
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const { email, password } = this.state;
+    // used to send email and pass to api
+    this.props.login(email, password);
+    // clear form
+    this.state = {
+      email: '',
+      password: '',
+    };
+  }
+
+  redirectToAdminPage() {
+    this.props.history.push('/admin');
+  }
+
+  render() {
+    const { loading, error, isAuthenticated } = this.props;
+    if (isAuthenticated) { this.redirectToAdminPage(); }
+    return (
+      <div className="container">
+        <div className="row justify-content-center">
```

Comment:
You can update the page title by using the `Helmet` component nested under the `.container` div:
```jsx
<Helmet>
  <title>Login</title>
</Helmet>
```